### PR TITLE
Updates to the create webhook fleetautoscaler guide.

### PR DESCRIPTION
**Pull request by roberthbailey**
Part of https://github.com/googleforgames/agones/issues/703.

### CODE REVIEW

File: site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md [hunk 1]
```diff
@@ -290,28 +316,29 @@ simple-udp-884fg-b7l58   Allocated   35.247.117.202   7766     minikube   5m
 You can delete the autoscaler service and associated resources with the following commands.
 
 ```
-kubeclt delete -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/autoscaler-webhook/autoscaler-service.yaml
+kubectl delete -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/examples/autoscaler-webhook/autoscaler-service.yaml
```
Comment (markmandel) [inline site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md]: Nice catch!

File: site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md [hunk 2]
```diff
@@ -7,8 +7,13 @@ description: >
...
 ---
 
-The main difference from the Buffer policy is that the logic on how many target replicas you need is delegated to a separate pod.
-This type of Autoscaler would send an HTTP request to the webhook endpoint every sync period (which is currently 30s) with a JSON body, and scale the target fleet based on the data that is returned.
+In some cases, your game servers may need to use custom logic for scaling your fleet that is more complex that what
+can be expressed using the Buffer policy in the fleetautoscaler. This guide shows how you can extend Agones
+with an autoscaler webhook to implement a custom autoscaling policy.
+
+When you use an autoscaler webhook the logic computing the number of target replicas is delegated to a
+separate pod. The fleetautoscaler will send an HTTP request to the webhook autoscaler's `/scale` endpoint every
```
Comment (markmandel) [inline site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md]: replace "seperate pod"  with something like "separate HTTP/s endpoint, such as one provided by a Kuebrnetes Deployment and Service within the same cluster. WDYT?

File: site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md [hunk 3]
```diff
@@ -290,28 +316,29 @@ simple-udp-884fg-b7l58   Allocated   35.247.117.202   7766     minikube   5m
...
 
 ## Chapter 2 Configuring HTTPS fleetautoscaler webhook with CA Bundle
 
 ### Objectives
 
-Using TLS and CA bundle we can establish trusted communication between Fleetautoscaler and Webhook which controls size of the fleet (Replicas count). The certificate of the webhook should be signed by Certificate Authority provided in fleetautoscaler yaml configuration file. Which eliminates the possibility to perform man in the middle attack when using HTTP connection to a webhook which can be located inside or outside of our cluster.
-
-Description of common steps with Chapter 1 would be omitted for simplicity, you can see previous chapter for the details.
+Using TLS and a certificate authority (CA) bundle we can establish trusted communication between Fleetautoscaler and
+the pod running the autoscaling webhook controlling size of the fleet (Replicas count). The certificate of the
```
Comment (markmandel) [inline site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md]: Same as above - "pod" - service maybe?

File: site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md (hunk 2 above)
Comment (roberthbailey) [reply site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md]: that sounds better, since it doesn't have to be running in a pod.

**Pull request merged**